lib/matplotlib/cbook/__init__.py

```diff
@@ -197,16 +197,17 @@ def __setstate__(self, state):
             s: {proxy: cid for cid, proxy in d.items()}
             for s, d in self.callbacks.items()}
 
-    def connect(self, s, func):
+    @_api.rename_parameter("3.3.4", "s", "signal")
```

Comment:
```suggestion
    @_api.rename_parameter("3.4", "s", "signal")
```
We only do API changes in minor releases not in bugfix releases. Since 3.4 is about to be tagged really soon, this may even need to go to "3.5".